components/support/sql/src/conn_ext.rs

```diff
@@ -70,7 +70,10 @@ pub trait ConnExt {
     }
 
     /// Execute a query that returns a single result column, and return that result.
-    fn query_one<T: FromSql>(&self, sql: &str) -> SqlResult<T> {
+    /// NOTE: rusqlite now has a builtin `query_one` (with not quite these semantics)
+    /// and a `one_column` (with these semantics but subtly different args) which should
+    /// generally be preferred. We've kept this to make upgrading easier.
+    fn conn_ext_query_one<T: FromSql>(&self, sql: &str) -> SqlResult<T> {
```

Comment:
This makes total sense to me.  Let's continue using this one for now, but try to move to one of the builtin methods.